### PR TITLE
Exercise 13

### PR DESCRIPTION
**Pull request by JackieMaw**
Adding Loggly

### CODE REVIEW

**Comment by JackieMaw** [comment]
Logs are showing in Loggly:
image

**Comment by JackieMaw** [comment]
I have Loggly working from Heroku and Azure. Once Exercise 12 PR has been approved I'll merge it over and then get Loggly working from Azure via Terraform.

File: README.md [hunk 1]
```diff
@@ -231,10 +226,19 @@ https://github.com/settings/applications/1847131
 
 PRODUCTION AZURE
 https://github.com/settings/applications/1857732
-        http://jackieu-todoapp.azurewebsites.net/
-        http://jackieu-todoapp.azurewebsites.net/login/callback
+        https://jackiemaw-do-me.azurewebsites.net/
+        https://jackiemaw-do-me.azurewebsites.net//login/callback
```
Comment (BenRamchandani) [inline README.md:230]:
```suggestion
        https://jackiemaw-do-me.azurewebsites.net/login/callback
```

**Review by BenRamchandani — APPROVED** [review]
Looks good, and I can see you've got a lot of existing logging. Perhaps too much in some cases, the authorization code and access token from the GitHub OAuth flow shouldn't be logged, except maybe at debug level.

**Pull request merged**